pages/docs/support/upgrading-grouparoo/v02-v03.mdx

```diff
@@ -0,0 +1,86 @@
+---
+title: "Upgrading Grouparoo from v0.2 to v0.3"
+date: "2021-04-16"
+pullQuote: "v0.3 is an exciting release with some breaking changes! Learn how to upgrade to v0.3."
+---
+
+Hello Grouparoo Community! v0.3 introduces the core concept of "Sync Modes" and lays the groundwork for adding sync modes across all current and future destinations. We only expect these breaking changes to affect a subset of our users.
+
+## Migration Guide
+
+**Upgrading Grouparoo**
+
+1. Update Grouparoo to the latest version through the usual steps by updating all the package version numbers to the latest version. More information is documented in [Installing updates](/docs/support/upgrading-grouparoo#installing-updates).
```

Comment:
Added some enhancements to that page via https://github.com/grouparoo/www.grouparoo.com/pull/381